src/common/utility.cpp

```diff
@@ -190,6 +190,13 @@ QByteArray Utility::userAgentString()
     return re.toLatin1();
 }
 
+QByteArray Utility::friendlyUserAgentString()
+{
+    const auto pattern = QStringLiteral("%1 (Desktop Client - %2)");
```

Comment:
So it would show "Linux (Desktop Client - [what here])"?

Could you show 1–2 examples of what it could show? :)

Reply:
Nope, It'd show "$Name (Desktop Client - $OS)". Like here it gives me "wintermute (Desktop Client - Linux)" and similar for other OSes.